spring-components/rdf4j-spring-demo/src/main/java/org/eclipse/rdf4j/spring/demo/dao/ArtistDao.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * ******************************************************************************
```

Comment:
This header looks odd - something went wrong with the comment annotation i think? Also noticed this in a couple of other files.

Reply:
It seems this is what happens when you copy/paste the rdf4j header into intellij's license profile. The best I can do with intellij (very helpful especially with new classes) is this:

```
/*
 * *****************************************************************************
 * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Distribution License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/org/documents/edl-v10.php.
 * *****************************************************************************
 */
```

Reply:
the whole `rdf4j-spring` module has this problem. Is it ok if I fix that in this PR?

Reply:
I'd really prefer if we could stick to the single, prescribed format for the license header. @hmottestad I know you are an IntelliJ user - can you advise on how you've configured it to use the correctly formatted copyright header? 

@fkleedorfer  as for fixing it in this in PR, that's not a problem.

Reply:
oh, no I meant, I'll make them look the way they are everywhere else, the intellij bit was just meant fyi. It would be nice of course if the formatter could be made to hammer the copyright into the required form, but it's not a problem - it's just a regex replace. So I'll fix that in all the places I touched.

Reply:
done